test/e2e/resize_nodes.go

```diff
@@ -560,6 +566,93 @@ var _ = Describe("Nodes [Disruptive]", func() {
 					}
 				}
 			})
+
+			// What happens in this test:
+			// 	Network connectivity on a node is cut off for 120 seconds to simulate network partition
+			// Expect to observe:
+			// 1. Node is marked NotReady after timeout by nodecontroller (40seconds)
+			// 2. All pods on node are marked NotReady shortly after #1
+			// 3. Node and pods return to Ready after connectivivty recovers
```

Comment:
That test should wait till all addons+static pods+pods on the node at the start of the test are ready, if it does not that is a bug. 

Regarding your pods (that you create in the test), you're not actually marking them ready right? you're just leaving it to the kubelet, so isn't this like any other kubelet-readiness test i.e is there already a test that checks that pods can successfully go from ready -> not ready -> ready today without the netsplit?